AutoResxTranslator/frmMain.Designer.cs

```diff
@@ -28,620 +28,676 @@ protected override void Dispose(bool disposing)
+            // 
+            this.label7.AutoSize = true;
+            this.label7.Location = new System.Drawing.Point(53, 20);
+            this.label7.Name = "label7";
+            this.label7.Size = new System.Drawing.Size(65, 17);
+            this.label7.TabIndex = 4;
+            this.label7.Text = "Resx File:";
+            // 
+            // tabBrowser
+            // 
+            this.tabBrowser.Controls.Add(this.webBrowser);
+            this.tabBrowser.Location = new System.Drawing.Point(4, 26);
+            this.tabBrowser.Name = "tabBrowser";
+            this.tabBrowser.Padding = new System.Windows.Forms.Padding(3);
+            this.tabBrowser.Size = new System.Drawing.Size(642, 342);
+            this.tabBrowser.TabIndex = 4;
+            this.tabBrowser.Text = "Google Translator";
+            this.tabBrowser.UseVisualStyleBackColor = true;
+            // 
+            // webBrowser
+            // 
+            this.webBrowser.AllowWebBrowserDrop = false;
+            this.webBrowser.Dock = System.Windows.Forms.DockStyle.Fill;
+            this.webBrowser.IsWebBrowserContextMenuEnabled = false;
+            this.webBrowser.Location = new System.Drawing.Point(3, 3);
+            this.webBrowser.MinimumSize = new System.Drawing.Size(20, 20);
+            this.webBrowser.Name = "webBrowser";
+            this.webBrowser.Size = new System.Drawing.Size(636, 336);
+            this.webBrowser.TabIndex = 0;
+            this.webBrowser.Url = new System.Uri("https://translate.google.com/", System.UriKind.Absolute);
+            this.webBrowser.DocumentCompleted += new System.Windows.Forms.WebBrowserDocumentCompletedEventHandler(this.webBrowser_DocumentCompleted);
+            // 
+            // lnkAbout
+            // 
+            this.lnkAbout.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Right)));
+            this.lnkAbout.AutoSize = true;
+            this.lnkAbout.Location = new System.Drawing.Point(621, 13);
+            this.lnkAbout.Name = "lnkAbout";
+            this.lnkAbout.Size = new System.Drawing.Size(45, 17);
+            this.lnkAbout.TabIndex = 3;
+            this.lnkAbout.TabStop = true;
+            this.lnkAbout.Text = "About";
+            this.lnkAbout.LinkClicked += new System.Windows.Forms.LinkLabelLinkClickedEventHandler(this.lnkAbout_LinkClicked);
+            // 
+            // tabPage3
+            // 
+            this.tabPage3.Controls.Add(this.groupBox2);
+            this.tabPage3.Location = new System.Drawing.Point(4, 26);
+            this.tabPage3.Name = "tabPage3";
+            this.tabPage3.Padding = new System.Windows.Forms.Padding(3);
+            this.tabPage3.Size = new System.Drawing.Size(642, 342);
+            this.tabPage3.TabIndex = 5;
+            this.tabPage3.Text = "Translate Service";
+            this.tabPage3.UseVisualStyleBackColor = true;
+            // 
+            // groupBox2
```

Comment:
groupBox2 should have anchor

Reply:
Haven't worked much on WinForms, so I still would have missed this.
Thank you.